### ISSUE TITLE
`Rails/Blank` chokes on code which contains invokations of `#present?` without an explicit receiver

### ISSUE THREAD

**Issue by smaximov** [bbatsov/rubocop#4241]
ActiveSupport Core Extensions define `#blank?` and `#present?` on *every* object, that means they can be called without an explicit receiver (e.g., inside an instance method definition).

The following snippet demonstrates the problem:

``` ruby
def on_send
  return if !present?
end

def on_if
  return unless present?
end
```

## Expected behavior

Rubocop emits relevant warnings.

## Actual behavior

Rubocop fails with the following backtrace:

``` $ rubocop -Rd
For /private/tmp/blank: configuration from /Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/config/default.yml
Inheriting configuration from /Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/config/enabled.yml
Inheriting configuration from /Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/config/disabled.yml
Inspecting 1 file
Scanning /private/tmp/blank/blank.rb
An error occurred while Rails/Blank cop was inspecting /private/tmp/blank/blank.rb:2:12.
undefined method `source' for nil:NilClass
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/blank.rb:72:in `block in on_send'
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/blank.rb:61:in `not_present?'
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/blank.rb:69:in `on_send'
[... SNIP ...]
/Users/smaximov/.rbenv/versions/2.3.3/bin/rubocop:22:in `load'
/Users/smaximov/.rbenv/versions/2.3.3/bin/rubocop:22:in `<main>'
An error occurred while Rails/Blank cop was inspecting /private/tmp/blank/blank.rb:6:2.
undefined method `source' for nil:NilClass
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/blank.rb:99:in `block in on_if'
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/blank.rb:63:in `unless_present?'
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/blank.rb:93:in `on_if'
[... SNIP ...]
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop
/Users/smaximov/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/rubocop-0.48.1/lib/rubocop
/Users/smaximov/.rbenv/versions/2.3.3/bin/rubocop:22:in `load'
/Users/smaximov/.rbenv/versions/2.3.3/bin/rubocop:22:in `<main>'
C

Offenses:

blank.rb:2:3: C: Style/NegatedIf: Favor unless over if for negative conditions.
  return if !present?
  ^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected

2 errors occurred:
An error occurred while Rails/Blank cop was inspecting /private/tmp/blank/blank.rb:2:12.
An error occurred while Rails/Blank cop was inspecting /private/tmp/blank/blank.rb:6:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.48.1 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin16)
Finished in 0.17423499998403713 seconds
```

## Steps to reproduce the problem

Run `rubocop --rails --debug` on the snippet above.

## Workaround

Handle nil values of `receiver` in [RuboCop::Cop::Rails::Blank#on_send](https://github.com/bbatsov/rubocop/blob/33cda4978a3576e93d69c301e3a764a9b7e4c0d7/lib/rubocop/cop/rails/blank.rb#L69-L73) and [RuboCop::Cop::Rails::Blank#on_if](https://github.com/bbatsov/rubocop/blob/33cda4978a3576e93d69c301e3a764a9b7e4c0d7/lib/rubocop/cop/rails/blank.rb#L93-L101).

## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin16)
```

**Comment by smaximov** [comment]
Apparently, `Rails/Present` is also subject to this issue.

**Comment by bbatsov** [comment]
//cc @rrosenblum

**Comment by rrosenblum** [comment]
Thanks for the bug report. This should be a straight forward fix.
